charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.9.5
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.53.10
+version: 5.54.0
 home: https://github.com/argoproj/argo-helm
 icon: https://argo-cd.readthedocs.io/en/stable/assets/logo.png
 sources:
```

Comment:
Please update changelog
Ref: https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog

Reply:
![Screenshot 2024-01-30 at 12 22 05](https://github.com/argoproj/argo-helm/assets/33663870/f8ad8e4c-455c-48b3-8e24-575e406aecec)
Am i missing anything?

Reply:
Oh, when I posted the request, the diff wasn't shown.
Now no problem.